doc/IOTRACER.md

```diff
@@ -0,0 +1,304 @@
+# Standalone Linux IO Tracer Case Study
+
+## Introduction
+
+Standalone Linux IO Tracer (iotrace) is a tool for block device I/O tracing
+and management of created traces. In this document you will find out detailed
+description how to use it and a few examples of collected traces analytics.
+
+From high-level perspective, for each I/O to target device(s) basic metadata
+information is captured (IO operation type, address, size), supplemented with
+extended classification. Extended classification contains information about I/O
+type (direct / filesystem metadata / file) and target file attributes(e.g. file
+size). Afterwards traces can later be converted to JSON or CSV format and
+consumed by any further analytics. iotrace does it and extends outputs with
+post-processing fields like IO latency, IO queue depth, and more.
+
+iotrace is based on [Open CAS Telemetry Framework (OCTF)](https://github.com/Open-CAS/open-cas-telemetry-framework). Collected traces are stored in format defined by OCTF using google protocol
+buffer description. You can find this definition [here](https://github.com/Open-CAS/open-cas-telemetry-framework/blob/master/source/octf/proto/trace.proto). Such trace format allows to analyze
+it by any kind of tool like Java, C++, Python, etc.,
```

Comment:
I suggest "any kind of tool written in Java, C++, Python, or other languages( supported by Protocol Buffer)"